flume/doc/devel/add_new_sink.md

```diff
@@ -143,4 +143,4 @@ Sometimes all the necessary logic to persist the notified context data cannot be
 ## Contact information
 Francisco Romero Bueno (francisco.romerobueno@telefonica.com)
 <br>
-Fermín Galán Márquez (fermin.galanmarquez@telefonica.com) 
\ No newline at end of file
+Fermín Galán Márquez (fermin.galanmarquez@telefonica.com) 
```

Comment:
Appartently there is no change, but it seems that the develop file version doesn't have the ending newline. After this PR, it would have it.

NTC (just informative)